.github/docker/testing/Dockerfile.testing-plugins-alma9

```diff
@@ -0,0 +1,69 @@
+ARG REGISTRY_URL
+
+FROM ${REGISTRY_URL}/almalinux:9
+
+RUN bash -e <<EOF
+
+dnf install -y 'dnf-command(config-manager)' epel-release zstd jq
+dnf config-manager --set-enabled crb
+dnf clean all
+
+# Install Robotframework
+dnf install -y python3.11 python3.11-pip
+pip3.11 install robotframework robotframework-examples
+# Install snmpsim
+pip3.11 install snmpsim-lextudio
+
+# Install node
+curl -fsSL https://rpm.nodesource.com/setup_21.x | bash -
+yum install -y nodejs
+# Install mockoon
+npm install -g -D @mockoon/cli@7.0.0
+
+# Add Centreon plugins repositories
+echo -e '[centreon-plugins-stable]\n\
+name=centreon plugins stable x86_64\n\
+baseurl=https://packages.centreon.com/rpm-plugins/el9/stable/x86_64\n\
+enabled=1\n\
+gpgcheck=1\n\
+gpgkey=https://yum-gpg.centreon.com/RPM-GPG-KEY-CES\n\
+[centreon-plugins-stable-noarch]\n\
+name=centreon plugins stable noarch\n\
+baseurl=https://packages.centreon.com/rpm-plugins/el9/stable/noarch\n\
+enabled=1\n\
+gpgcheck=1\n\
+gpgkey=https://yum-gpg.centreon.com/RPM-GPG-KEY-CES\n\
+[centreon-plugins-testing]\n\
+name=centreon plugins testing x86_64\n\
+baseurl=https://packages.centreon.com/rpm-plugins/el9/testing/x86_64\n\
+enabled=1\n\
+gpgcheck=1\n\
+gpgkey=https://yum-gpg.centreon.com/RPM-GPG-KEY-CES\n\
+[centreon-plugins-testing-noarch]\n\
+name=centreon plugins testing noarch\n\
+baseurl=https://packages.centreon.com/rpm-plugins/el9/testing/noarch\n\
+enabled=1\n\
+gpgcheck=1\n\
+gpgkey=https://yum-gpg.centreon.com/RPM-GPG-KEY-CES\n\
+[centreon-plugins-unstable]\n\
+name=centreon plugins unstable x86_64\n\
+baseurl=https://packages.centreon.com/rpm-plugins/el9/unstable/x86_64\n\
+enabled=1\n\
+gpgcheck=1\n\
+gpgkey=https://yum-gpg.centreon.com/RPM-GPG-KEY-CES\n\
+[centreon-plugins-unstable-noarch]\n\
+name=centreon plugins unstable noarch\n\
+baseurl=https://packages.centreon.com/rpm-plugins/el9/unstable/noarch\n\
+enabled=1\n\
+gpgcheck=1\n\
+gpgkey=https://yum-gpg.centreon.com/RPM-GPG-KEY-CES\n'\
+>> /etc/yum.repos.d/centreon-plugins.repo
+
+mkdir -p /var/lib/centreon/centplugins/
+chmod 777 /var/lib/centreon/centplugins/
+
+dnf clean all
+
+EOF
+
+WORKDIR /tests
```

Comment:
@Evan-Adam if this is useful in this dockerfile, why not spread it in others?

Reply:
removed.